style.css

```diff
@@ -48,6 +48,11 @@ a:active {
 	text-decoration: none;
 }
 
+.wp-block-navigation .wp-block-navigation-item a:hover {
```

Comment:
Are you intentionally leaving out the focus state here, since it could look a little strange?

For example, when the sub-menu item is in focus:
<img width="382" alt="Screen Shot 2021-12-07 at 3 53 04 PM" src="https://user-images.githubusercontent.com/5375500/145104529-897a9434-9282-495c-af2b-33066006c07b.png">

Reply:
Ah yes, we should add this to focus too. I think it looks fine: 

![focus](https://user-images.githubusercontent.com/1202812/145241494-9336ee64-f0e7-4d37-8827-ef16b5e74ff7.gif)